Scripts/ReadPDFFileV2/ReadPDFFileV2.yml

```diff
@@ -128,6 +128,6 @@ subtype: python3
 runas: DBotWeakRole
 runonce: false
 tests:
-- Extract Indicators From File - test
+- Extract Indicators From File - Generic v2 - Test
```

Comment:
Why are we changing this?

Reply:
This test is using "Extract Indicators From File - Generic" playbook, which calls the readPdfFile script (the old one).
This means that until now this test didn't even use the readPdfFileV2 script.
Thought to update the used playbook to "Extract Indicators From File - Generic v2",
but after consulting @idovandijk we decided that it will be better to use "Extract Indicators From File - Generic v2 - Test" which covers more cases (and of-course using the new readPdfFile script).